gap/oper.gi

```diff
@@ -1771,3 +1771,67 @@ function(digraph, k)
 
   return DigraphByAdjacencyMatrix(out);
 end);
+
+#
+
+InstallMethod(DIGRAPHS_IsMatching,
```

Comment:
Is `DIGRAPHS_IsMatching` the best name for this?  Since it returns more information than just `true` or `false`, how about `DIGRAPHS_MatchingSeen` or just `DIGRAPHS_Matching`?

This is non-user-facing though, so I'm not too bothered.